hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -60,18 +60,18 @@
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private volatile boolean updateRequested = false;
-    private volatile boolean updating = false;
-
     @EJB
     PropertiesService properties;
 
     @EJB
     DefinitionsService definitions;
 
+    @Resource(lookup = "java:jboss/infinispan/cache/hawkular-alerts/dataIds")
```

Comment:
Maybe add a comment here explaining how this cache is used.  Maybe mention that the tenantId in the key is applicable to the value (so the values can just be a Set of dataIdsIds).

Suggestion: dataIdsCache -> publishDataIdsCache, to better reflect it's part of the publishing mechanism.